### PR TITLE
display assets created by user with 0 balance

### PR DESCRIPTION
**Pull request by anakin001**

### CODE REVIEW

File: src/modules/account/views/ViewAccountAssets.vue [hunk 1]
```diff
@@ -116,6 +119,49 @@ export default {
...
+        const fetchNoBalanceMosaics = async()=>{
+            if(!acc.value){
+                return 
+            }
+            let txnAPI = AppState.chainAPI.transactionAPI
+            let txnQueryParams = new TransactionQueryParams
+            txnQueryParams.embedded = true
+            txnQueryParams.type = [TransactionType.MOSAIC_DEFINITION]
+            txnQueryParams.publicKey = publicAccount
+            let searchedTxn = await txnAPI.searchTransactions(TransactionGroupType.CONFIRMED,txnQueryParams)
```
Comment (shinneng) [inline src/modules/account/views/ViewAccountAssets.vue:133]: I think it's better to use the new mosaic endpoint.  you can start testing the new endpoint at "api-1.testnet2.xpxsirius.io:3001".  The new rest will be deployed to testnet and mainnet soon

**Review by shinneng — CHANGES_REQUESTED** [review]

File: src/modules/account/views/ViewAccountAssets.vue [hunk 2]
```diff
@@ -109,6 +112,47 @@ export default {
...
+                        isCreator: true
+                    });
+                }
+            })
+            let mosaicIds = []
+            noBalanceMosaics.value.forEach(mosaic=>{
+                mosaicIds.push(new MosaicId(mosaic.id))
+            })
+            let mosaicNames = await AppState.chainAPI.assetAPI.getMosaicsNames(mosaicIds)
+            noBalanceMosaics.value.forEach((mosaic,index)=>{
```
Comment (sleepyOwl14) [inline src/modules/account/views/ViewAccountAssets.vue:142]: wrong approach, please wait for new PR
- missing pagination and loading
- missing asset name, data loading in batch, limit number per request

**Review by sleepyOwl14 — CHANGES_REQUESTED** [review]
please wait for new PR, then rewrite the 0 balance UI flow

File: src/modules/account/views/ViewAccountAssets.vue (hunk 2 above)
Comment (anakin001) [reply src/modules/account/views/ViewAccountAssets.vue:142]: ok noted, thanks

**Pull request closed without merging**